pkgs/development/python-modules/cartopy/default.nix

```diff
@@ -28,17 +30,17 @@ buildPythonPackage rec {
     export HOME=$(mktemp -d)
     ${maybeXvfbRun} pytest --pyargs cartopy \
       -m "not network and not natural_earth" \
-      -k "not test_nightshade_image and not background_img"
+      -k "not test_nightshade_image and not background_img and not test_eccentric_globe and not test_ellipse_globe"
   '';
```

Comment:
Please use pytestCheckHook.

Reply:
I was afraid thus would not work with the `xvfb-run` wrapper, but since this is py2.7 only, and that cartopy is py-3+ only, I just drop that alltogether and migrated to `pytestCheckHook`.

Thanks!